editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -321,19 +321,15 @@ const RowForArrayControl = betterReactMemo(
             {propertyStatus.overwritable ? (
               <SquareButton highlight onMouseDown={toggleInsertRow}>
                 {insertingRow ? (
-                  <Icn
+                  <Icons.Minus
```

Comment:
Was this change supposed to be in this PR?

Reply:
I initially thought we'd use the defaults on these, but in general I'm trying to use the `Icons` object, makes it easier to see where we're (not) being consistent.